migration-services/pom.xml

```diff
@@ -61,6 +61,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-entitymanager</artifactId>
+      <version>4.2.21.Final</version>
```

Comment:
The version is defined in the dependencies pom, it should not define again here.